src/material-examples/button-card/button-card-example.html

```diff
@@ -0,0 +1,9 @@
+<mat-card>
+  <mat-card-header>
+    <mat-card-title>Card with action buttons</mat-card-title>
+  </mat-card-header>
+  <mat-card-actions align="end">
+    <button mat-button>CANCEL</button>
```

Comment:
Use normal text case for buttons, the docs site capitalizes them via CSS. If screen-readers encounter ALL CAPS, it reads it as "A. L. L. C. ..."

(here and elsewhere)